hubblestack/extmods/modules/win_pulsar_winaudit.py

```diff
@@ -232,6 +246,7 @@ def _check_acl(path, mask, wtype, recurse):
     else:
         wtype = [wtype]
 
+    path = "'" + path + "'"
```

Comment:
lost in commit https://github.com/hubblestack/hubble/commit/7cccf7b9e92dfbc32c0253bc3d63418962852dcd